app/models/measure.rb

```diff
@@ -266,10 +266,6 @@ def national?
     national
   end
 
-  def invalidated?
```

Comment:
@alexey-klimuk is this method not called? so can be removed?

Reply:
@matthewford  it's duplicated